presets.js

```diff
@@ -18,1099 +20,1444 @@ exports.getPresets = function () {
 
-	let presets = [
-		{
+	let presets = {
+		playSelected: {
+			type: 'button',
 			category: 'Playlist',
-			label: 'Play Selected',
-			bank: {
-				style: 'png',
+			name: 'Play Selected',
+			options: [],
+			style: {
 				png64: playIcon,
 				pngalignment: 'center:center',
 				size: '18',
 				color: ColorWhite,
 				bgcolor: ColorBlack,
 			},
-			actions: [
+			steps: [
 				{
-					action: 'play_select',
+					down: [{ actionId: 'play_select' }],
+					up: [],
 				},
 			],
+			feedbacks: [],
 		},
-		{
+		playPlaylist: {
+			type: 'button',
 			category: 'Playlist',
-			label: 'Play Playlist',
-			bank: {
-				style: 'png',
+			name: 'Play Playlist',
+			style: {
 				png64: playPlaylistIcon,
 				pngalignment: 'center:center',
 				size: '14',
 				color: ColorWhite,
 				bgcolor: ColorBlack,
 			},
-			actions: [
+			steps: [
 				{
-					action: 'play',
+					down: [
+						{
+							actionId: 'play',
+						},
+					],
+					up: [],
 				},
 			],
+			feedbacks: [],
 		},
-
-		{
+		pause: {
+			type: 'button',
 			category: 'Playlist',
-			label: 'Pause',
-			bank: {
-				style: 'png',
+			name: 'Pause',
+			style: {
 				png64: pauseIcon,
 				pngalignment: 'center:center',
 				size: '14',
 				color: ColorWhite,
 				bgcolor: ColorBlack,
 			},
-			actions: [
+			steps: [
 				{
-					action: 'stop',
+					down: [
+						{
+							actionId: 'stop',
+						},
+					],
+					up: [],
 				},
 			],
+			feedbacks: [],
 		},
-
-		{
+		jumpPrevious: {
+			type: 'button',
 			category: 'Playlist',
-			label: 'Jump to previous',
-			bank: {
-				style: 'png',
+			name: 'Jump to previous',
+			style: {
 				png64: skipBackwardIcon,
 				pngalignment: 'center:center',
 				size: '14',
 				color: ColorWhite,
 				bgcolor: ColorBlack,
 			},
-			actions: [
+			steps: [
 				{
-					action: 'jump_prev',
+					down: [
+						{
+							actionId: 'jump_prev',
+						},
+					],
+					up: [],
 				},
 			],
+			feedbacks: [],
 		},
-
-		{
+		jumpNext: {
+			type: 'button',
 			category: 'Playlist',
-			label: 'Jump to next',
-			bank: {
-				style: 'png',
+			name: 'Jump to next',
+			style: {
 				png64: skipForwardIcon,
 				pngalignment: 'center:center',
 				size: '14',
 				color: ColorWhite,
 				bgcolor: ColorBlack,
 			},
-			actions: [
+			steps: [
 				{
-					action: 'jump_next',
+					down: [
+						{
+							actionId: 'jump_next',
+						},
+					],
+					up: [],
 				},
 			],
+			feedbacks: [],
 		},
-
-		{
+		pauseResume: {
+			type: 'button',
 			category: 'Playlist',
-			label: 'Pause / Resume',
-			bank: {
-				style: 'png',
+			name: 'Pause / Resume',
+			style: {
 				png64: playPauseIcon,
 				pngalignment: 'center:center',
 				size: '14',
 				color: ColorWhite,
 				bgcolor: ColorBlack,
 			},
-			actions: [
+			steps: [
 				{
-					action: 'toggle_play',
+					down: [
+						{
+							actionId: 'toggle_play',
```

Comment:
Do these presets work? I have a feeling they will error without an options property, even if it is empty

Reply:
They do work, but I'll go ahead and add an empty options property to be safe / future proof

Reply:
Addressed with https://github.com/bitfocus/companion-module-imimot-mitti/pull/25